datastore/schedule-export/cron.yaml

```diff
@@ -0,0 +1,4 @@
+cron:
+- description: "Daily Cloud Datastore Export"
+  url: /cloud-datastore-export?output_url_prefix=gs://my-export-bucket&kind=task
```

Comment:
s/my-export-bucket/"<MY-EXPORT-BUCKET" 

so it is clear this is a value that needs to be replaced. (Make sure to mention in README)